### ISSUE TITLE
Worksheet Id changing after reopening workbook on different Excel 2016 Version

### ISSUE THREAD

**Issue by soorajhealthplotter** [OfficeDev/office-js#1390]
## Expected Behavior
The worksheet id should be unique across different version of Excel(Office 365,Office online,Office 2016)

## Current Behavior
I have worksheet id is "{BF289900-56FF-4681-92AD-4C1C8EC2CB39}", then I created some sheet specific metadata to store data in database using worksheet id and saved workbook. I send the file over the client and the client was not able to access metadata. When debugging i understood the worksheet id got changed it shows me that the worksheet.id now is "{00000000-0001-0000-0003-000000000000}" . I created the file in Office 365 and the client opened the file in Office 2016.
From the documentation i understood that for a workbook the worksheet id is going to be unique

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Online
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Browser (if using Office Online): Chrome

## Client Environment

* Platform [PC desktop, Mac, iOS, Office Online]: PC Desktop
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: Microsoft Office Professional Plus 2016
* Operating System: Windows

**Comment by soorajhealthplotter** [comment]
I think the issue can be related to https://github.com/OfficeDev/office-js-docs/issues/297

**Comment by lumine2008** [comment]
Thanks @soorajhealthplotter for reporting this issue, it seems we are not be able to repro this issue, would you please use the script lab to generate the gist and I could repro this issue in our side.
![image](https://user-images.githubusercontent.com/26398035/93727632-8deb0780-fbee-11ea-9ae6-ecd18365f83e.png)

**Comment by soorajhealthplotter** [comment]
Actually my issue is when i open the same workbook in different version of excel in different system the worksheet id is different. i am not sure how can gist reproduce issue 

function run() {
  return Excel.run(function(context) {
    var range = context.workbook.worksheets.getActiveWorksheet();
    range.load("id");
    return context.sync().then(function() {
      console.log('The worksheet id is "' + range.id + '".');
    });
  });
}

running this code in excel 365 gives me "{BF289900-56FF-4681-92AD-4C1C8EC2CB39}" as worksheet id and in office 2016 the the worksheet id changes to "{00000000-0001-0000-0200-000000000000}"

**Comment by lumine2008** [comment]
We have tried O365 and Excel online but not repro. what's your version? kindly please share a bit more information for us to repro. thanks

**Comment by soorajhealthplotter** [comment]
I am still having issues. for office 365 and excel online iam not having issues,but is excel 2016 ia am having issue

**Comment by soorajhealthplotter** [comment]
Platform [PC desktop, Mac, iOS, Office Online]: PC Desktop
Host [Excel, Word, PowerPoint, etc.]: Excel
Office version number: Microsoft Office Professional Plus 2016
Operating System: Windows

**Comment by soorajhealthplotter** [comment]
I have worksheet id is "{BF289900-56FF-4681-92AD-4C1C8EC2CB39}", then I created some sheet specific metadata to store data in database using worksheet id and saved workbook.

I send the file over the client and the client was not able to access metadata. When debugging i understood the worksheet id got changed it shows me that the worksheet.id now is "{00000000-0001-0000-0003-000000000000}" . 

I created the file in Office 365 and the client opened the file in Office 2016.

From the documentation i understood that for a workbook the worksheet id is going to be unique
